qa/tasks/vstart_runner.py

```diff
@@ -717,6 +717,12 @@ def _get_mount_cmd(self, mntopts, mntargs):
 
         if os.getuid() != 0:
             mount_cmd += ['--client_die_on_failed_dentry_invalidate=false']
+
+        # XXX: Passing ceph-fuse option above makes using sudo command
+        # redunant. Plus, we prefer that vstart_runner.py doesn't use sudo
+        # command as far as possbile.
+        mount_cmd.remove('sudo')
```

Comment:
Shouldn't this be under the `if` block above?

Reply:
Fixed